.gitignore

```diff
@@ -254,4 +254,8 @@
 compile_flags.txt
 
 # Auto-generated resource script
-resources/environment.sh
\ No newline at end of file
+resources/environment.sh
+
+# Keyman translations from https://crowdin.com/project/keyman
+Keyman.zip
```

Comment:
I'd prefer to avoid a generic filename like this in the root of the project. Is it possible to download to a .crowdin-tmp/ folder or something like that?